.github/workflows/deploy-feed-discovery-image.yaml

```diff
@@ -0,0 +1,26 @@
+name: build-and-push-feed-discovery
+
+on:
+  push:
+    branches:
+      - 'issue#1743'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: src\api\feed-discovery
+          dockerfile: Dockerfile
+          push: true
+          tags: user/app:latest
```

Comment:
Let's tag this slightly different, so we update the `latest` tag for `feed-discovery` to this new image:

```
tags: feed-discovery:latest
```

I also wonder if we should use `feed-discovery:${{github.sha}}` later on.  But we can add that if we need to go back in time to get old versions.